src/tests/localization/localization.spec.ts

```diff
@@ -0,0 +1,10 @@
+import { test } from '../../fixtures';
+import { getLocalizationData } from '../../utils/csvReader';
+
+getLocalizationData().forEach(({ languageName, headerKeywords, footerKeywords }) => {
+    test(`should verify header and footer in ${languageName}`, async ({ calculatorPage, layoutPage }) => {
+        await calculatorPage.open();
+        await layoutPage.expectHeaderContains(headerKeywords);
```

Comment:
I would recommend verifying the element by the provided date strictly instead of contains